codestodians/food-basket-front-end
# [Feature]: Refactor Signup Component

#### What does this PR do?
- Updates the `SignUpForm` to use `react-apollo`'s `Mutation` component.

#### Description of Task to be completed?
- Update `react-apollo`.
- Refactor the `SignUpForm` component to use the `Mutation` component from `react-apollo`.
- Deprecate the `SignupForm-connect` module.

#### How should this be manually tested?
- Check out into the `feature/mutation-component-for-signup-158070194` branch.
- Install dependencies.
- Run the App and test that Signup works as before.
- Run tests to see that all tests pass.

#### What are the relevant pivotal tracker stories?
- [#158070194](https://www.pivotaltracker.com/story/show/158070194)
